### ISSUE TITLE
CMakeLists.txt: osi_version.proto directory bug for flatbuf build

### ISSUE THREAD

**Issue by thomassedlmayer** [OpenSimulationInterface/open-simulation-interface#704]
#### Describe the bug
The OSI flatbuffers build does not run properly with the current CMakeLists.txt because for the osi_version.proto both CMAKE_CURRENT_SOURCE_DIR and CMAKE_CURRENT_BUILD_DIR are added to the path. Only the build directory should be used for osi_version.proto.

#### Describe how to reproduce the bug
1. CMakeLists.txt (16): Set option(OSI_BUILD_FLATBUFFER "Build flatbuffer versions of libraries" ON)
2. Try to build OSI
3. See error: 
```
Convert C:/Users/tsedlmayer/Documents/dev/open-simulation-interface/flatbuild/osi_version.proto to osi_version.fbs using flatc
C:\Users\tsedlmayer\Documents\dev\open-simulation-interface\flatbuild\flatbuffers-build\Release\flatc.exe : error : unable to load file: C:/Users/tsedlmayer/Documents/dev/open-simulation-interface/C:/Users/tsedlmayer/Documents/dev/open-simulation-interface/flatbuild/osi_version.proto
```

**Comment by pmai** [comment]
Will be fixed by #730

**Comment by jdsika** [comment]
Closed as completed
